### ISSUE TITLE
Missing a "Skip Navigation" Feature

### ISSUE THREAD

**Issue by wesleyboar** [TACC/Core-CMS#1125]
**Describe the bug**
Website does not have [a skip navigation link](https://webaim.org/techniques/skipnav/).

**To Reproduce**
2. Tab to first focusable elements.
3. User must tab through all navigation to reach main content.

**Expected behavior**
User should be able to [skip navigation](https://webaim.org/techniques/skipnav/).

**Screenshots**
https://github.com/user-attachments/assets/2b44addc-7b4b-4879-af7f-d97a627059f6
